react/__tests__/components/__snapshots__/Container.test.js.snap

```diff
@@ -3,7 +3,7 @@
 exports[`<Container /> component should match snapshot 1`] = `
 <DocumentFragment>
   <section
-    class="container ph3 ph5-m ph2-xl mw9 center"
+    class="ph3 ph5-m ph2-xl mw9 center"
```

Comment:
Because we're now using the HOC `withCssHandles` we won't have the `container` class in the snapshot anymore, since there is no runtime to generate this

Reply:
Maybe we should mock the withCssHandles to return an object with that name.